### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
1)this is live demo <a href="https://raw.githack.com/beyk/smashing_magazine_clone/feature/index.html">click here</a>
2)this webpage was used as model webpage <a href="https://www.smashingmagazine.com/">click here</a>
3)modern web design techniques like grid and flexbox were used to style the page.
** feature branch is used for this pull request**

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Thank you for implementing the changes @beyk 💪🙌.

### But there are some small issues that are preventing me from approving this project.

- There is still a horizontal scrollbar and the same footer elements are sticking out, as I pointed out earlier.

  Annotation 2020-06-16 092733

- There should be two placeholders for the footer images. You have only placed one.

   Right now 👇

   Annotation 2020-06-16 093236

   Original 👇

   image

I understand that you are doing this project for a long time and without a partner. I really appreciate your determination and resiliency 💪👌 and we are thankful to you for this. 

*I see that you are creating new PRs for every code request. From the next time request for code reviews from the same PR. It's a bad practice to open PRs for every code reviews. The reviewers might lose context and it creates confusion.*

### Please 🙏 make these changes and request another code review.

Happy coding 👨‍💻

**Comment by ghost** [comment]
1)this is live demo <a href="https://raw.githack.com/beyk/smashing_magazine_clone/feature/index.html">click here</a>
2)this webpage was used as model webpage <a href="https://www.smashingmagazine.com/">click here</a>
3)modern web design techniques like grid and flexbox were used to style the page.
** feature branch is used for this pull request**

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: index.html [hunk 1]
```diff
@@ -1,37 +1,497 @@
...
+            <div class="singing-barista"></div>
+            <h2>1.999 smashing member</h2>
+            <p class="meow__intro">A friendly community that keeps this website alive. With books, webinars, discounts and networking — for just 1 coffee a month. <a href="/membership/" class="smashing-member-family">Join the family.</a></p>
+            <a href="/membership" class="btn">Explore features&nbsp;→</a>
+        </div>
+        <div class="load-more"><a class="
+            " href="#">See All <span class="p1">Latest Articles</span></a></div>
+    </section>
+    <section class="pre-release">
+        <div class="">
```
Comment (andresporras3423) [inline index.html]: You should use article tag, instead of div, in this part of your page

File: index.html [hunk 2]
```diff
@@ -1,37 +1,497 @@
...
+                            <a href="#" class="expo">46</a>
+                            <a href="#" class="base">#User Experience</a>
+                            <a href="#" class="expo">299</a>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+    </main>
+    <section class="latest_post">
```
Comment (andresporras3423) [inline index.html:107]: inside this section, replace div with article tag

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @jamestucker1988** 
Good job, your project looks good :muscle:  
Your page meets most of the requirements. :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [ ] Remember to use article tag when it's possible, check if a part of your code is a title followed by paragraphs, if so then use article tag.
- [ ]  Your page shouldn't have a horizontal scroll bar. Make sure that for any large size screen (1200px or bigger) your page doesn't generate this horizontal scrollbar:



Also, make sure to correct this:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

**Comment by ghost** [comment]
1)this is live demo <a href="https://raw.githack.com/beyk/smashing_magazine_clone/feature/index.html">click here</a>
2)this webpage was used as model webpage <a href="https://www.smashingmagazine.com/">click here</a>
3)modern web design techniques like grid and flexbox were used to style the page.
** feature branch is used for this pull request**

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Nice job implementing the changes 👌🙌👏

### However, one small issue is still preventing me from approving this project.

- There is still a horizontal scrollbar in your clone.
Annotation 2020-06-17 000537
 Check the comments for more details.

File: assets/styles/main.css [hunk 3]
```diff
@@ -1,49 +1,818 @@
 body {
-    
+  font-size: 1.125em;
+  background-color: #f6f3f2;
+  width: 100%;
```
Comment (tirthajyoti-ghosh) [inline assets/styles/main.css]: This is the root of the scrollbar problem.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply this small change and request for code review.

Happy coding 👨‍💻

**Comment by ghost** [comment]
1)this is live demo <a href="https://raw.githack.com/beyk/smashing_magazine_clone/feature/index.html">click here</a>
2)this webpage was used as model webpage <a href="https://www.smashingmagazine.com/">click here</a>
3)modern web design techniques like grid and flexbox were used to style the page.
** feature branch is used for this pull request*

File: README.md [hunk 4]
```diff
@@ -1,2 +1,62 @@
...
+- Github: [jamestucker1988](https://github.com/jamestucker1988)
+- Twitter: [anandku74070598](https://twitter.com/anandku74070598)
+- Linkedin:  [Anand Kumar](https://linkedin.com/in/anand-kumar-9128)
+
+​
+👤 **Ashgar beyk**
+​
+- Github: 
+- Twitter: 
+- Linkedin:
```
Comment (elmejdki) [inline README.md:43]: your partner has missing links for his social media profiles

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job so far @team :clap: :tophat:, but your project needs to have some changes in order to make it approved
### status: require changes :recycle: 

I noticed that you didn't fix the requirements mentioned by @tirthajyoti-ghosh please fix this issue:
image

### please fix all the required changes before you submit another pull request.
_If you have trouble to remove that horizontal scrollbar feel free to contact me on slack I will be happy to help you out_
### Happy Coding

**Comment by ghost** [comment]
1)this is live demo <a href="https://raw.githack.com/beyk/smashing_magazine_clone/feature/index.html">click here</a>
2)this webpage was used as model webpage <a href="https://www.smashingmagazine.com/">click here</a>
3)modern web design techniques like grid and flexbox were used to style the page.
** feature branch is used for this pull request*

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the changes requested by myself and the previous TSEs! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****
### [OPTIONAL]

- While your linters are showing no errors, they are showing some warnings. Feel free to use the "Webhint Report" in the linters checks and fix those warnings.
Screen Shot 2020-06-18 at 8 08 41 PM

- I noticed that this is not your first review. In your future projects, if you closed the original pull request by accident, you should add a link referencing the original pull request in the subsequent pull request. 😄

_P.S. all requested changes marked by [OPTIONAL] are not required. It is up to you whether or not you would like to do them_
****

###  Good Luck and Happy Coding! 🍀 👩🏾‍💻
_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Pull request merged**